### DISCUSSION TITLE
Lodestar Standup 2024-23 (June 18, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6893] [Standups]
### Meeting Info
[June 18, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/jun-18-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v2.0 Ideas - "Competitive Client" milestone (Performance strategy planning)
  - v1.20.0 Planning
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
We should definitely discuss what might be happening here and how to further investigate as complaints keep piling up: https://github.com/ChainSafe/lodestar/issues/6861

**Comment by twoeths** [comment]
I'd like to discuss deployment process/plan after a release is out

**Comment by philknows** [comment]
Closing for #6900
